search-app/src/cmr/search/services/association_validation.clj

```diff
@@ -0,0 +1,260 @@
+(ns cmr.search.services.association-validation
```

Comment:
I'm glad you were able to reuse a large chunk of code for both types of associations.